hmpps-prisoner-search-indexer/Running.md

```diff
@@ -1,125 +1,40 @@
 # Running Prisoner Search Indexer
 
-`localstack` is used to emulate the AWS SQS and OpenSearch service.
-
-#### Running prisoner search indexer in Docker
+`localstack` is used to emulate the AWS SQS, SNS and OpenSearch service.
 
 *WE STRONGLY ADVISE, WHEN RUNNING LOCALLY, TO POINT AT SERVICES IN THE T3/DEV ENVIRONMENT*
 
 Since the indexer has a number of dependencies that themselves have dependencies, it is recommended that when running
 locally you point mostly at services running in T3. The exception is pointing at localstack for AWS services,
 but HMPPS Auth, Prison Api, incentives services, restricted patients are best used directly on T3 dev environment.
-The docker-compose.yml file is provided as reference but the dependencies of dependencies are not shown and are subject to change.
-To run against T3 dependencies the environment variables as shown in the helm_deploy/values-dev.yaml file for paths can be used.
-Personal client credentials for the T3 services can be obtained from the HMPPS Auth team with the appropriate roles.
-
-To start up localstack and other dependencies with prisoner indexer running in Docker too:
-```shell
-docker compose up localstack hmpps-auth prison-api restricted-patients hmpps-incentives-api hpsi-db
```

Comment:
I've decided to remove most of this stuff since the dependencies don't fire up at present anyway - e.g. incentives requires localstack access and needs configuring

Reply:
I think that's fair enough - wonder if we should have a docker-compose-test.yml then?